tests/FileCard/index.js

```diff
@@ -25,8 +25,11 @@ describe('FileCard', () => {
 
   it('renders an element with the correct background-image', () => {
     expect(
-      wrapper.find('.file-card__thumbnail').props().style.backgroundImage
-    ).to.equal(`url(${props.previewSrc})`);
+      wrapper
+        .find('img')
+        .props()
+        .src.indexOf(props.previewSrc)
```

Comment:
@kyleharper used indexOf here to check the file URL as at the beginning of the attribute, is this the way to go? I am adding ?rand=Math.random to the URL so I can't be sure what the URL will be